### PR TITLE
feat(core): add the option to create a ShellbarUserMenu item with a glyph

### LINKED ISSUE #7327: [Enhancement] Ability to create UserMenu on ShellbarComponent with icons
#### Is this a bug, enhancement, or feature request?
Enhancement

#### Briefly describe your proposal.
Add Icon optional param on the `ShellbarUserMenu` interface and include the icon on the `shellbar-user-menu.component.html`

#### Which versions of Angular and Fundamental Library for Angular are affected? 
fundamental 0.32.3
ng 13.0.3

### PR DESCRIPTION
**Pull request by guisperandio**
## Related Issue(s)
closes #7327 

## Description
add the option to create a ShellbarUserMenu item with a glyph

### CODE REVIEW

File: libs/core/src/lib/shellbar/user-menu/shellbar-user-menu.component.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.fd-shellbar__icon {
```
Comment (mikerodonnell89) [inline libs/core/src/lib/shellbar/user-menu/shellbar-user-menu.component.scss:1]: We need to add this class to fundamental-styles, can you open a PR there as well?

**Review by mikerodonnell89 — CHANGES_REQUESTED** [review]
The icon should be changing to white as well when the item is active/pressed - see [here](https://fundamental-ngx.netlify.app/#/core/menu#with-icons)

Screen Shot 2021-12-07 at 11 49 49 AM

File: libs/core/src/lib/shellbar/user-menu/shellbar-user-menu.component.scss (hunk 1 above)
Comment (guisperandio) [reply libs/core/src/lib/shellbar/user-menu/shellbar-user-menu.component.scss:8]: It is the same "problem" that we have on the `ProductMenuComponent`, we have a specific use case for this `fd-icon` and we don't have an `fd-shellbar__icon` class on fundamental-styles

Comment (guisperandio) [reply libs/core/src/lib/shellbar/user-menu/shellbar-user-menu.component.scss]: I changed the fd-icon for the fd-menu-addon and I was able to remove the CSS file

**Review by platon-rov — APPROVED** [review]

**Review by dimamarksman — APPROVED** [review]

**Review by Betrozov — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Comment by guisperandio** [comment]
Waiting for @mikerodonnell89 re-review

**Review by mikerodonnell89 — APPROVED** [review]

**Comment by mikerodonnell89** [comment]
@guisperandio looks good 👍

**Comment by guisperandio** [comment]
> @guisperandio looks good 👍

@mikerodonnell89  I signed the CLA a couple of times, but it still shows that it is not signed, what should I do?

**Comment by guisperandio** [comment]
Is it possible to reopen this PR?

**Pull request closed without merging**